kibble/api/languages.go

```diff
@@ -20,6 +20,8 @@ import (
 	"kibble/models"
 	"sort"
 	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
I doubt we need to include this package. Even the one other usage that I can see in the codebase could be removed now with built-in error wrapping since 1.13.

Reply:
This was an auto-import on save... I wonder if I've stuffed up my Go env somehow?